paasta_tools/cli/cmds/local_run.py

```diff
@@ -622,6 +622,8 @@ def get_local_run_environment_vars(instance_config, port0, framework):
         env["MARATHON_APP_LABELS"] = ""
         env["MARATHON_APP_ID"] = "/simulated_marathon_app_id"
         env["MARATHON_HOST"] = hostname
+        env["PAASTA_HOST"] = hostname
```

Comment:
Actually, what's the content of PAASTA_HOST gonna be? The hostname of the physical machine?
Or the hostname that we use currently which is the concatenation of the machine hostname and some other sha?

Reply:
Nice catch. I thought the hostname here is the real one.

Reply:
Yeah the one that I want is the real one. There's already the HOST variable that contains the concatenation.